enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/StateMachineConfig.java

```diff
@@ -317,6 +317,15 @@ private void configurePresenceCheckTransitions(StateMachineTransitionConfigurer<
                 .action(presenceCheckInitAction)
                 .target(OnboardingState.PRESENCE_CHECK_IN_PROGRESS)
 
+                .and()
+                .withExternal()
+                .source(OnboardingState.PRESENCE_CHECK_IN_PROGRESS)
```

Comment:
The main change! Allow to send `PRESENCE_CHECK_INIT` event during `PRESENCE_CHECK_IN_PROGRESS`, respectively to call `/api/identity/presence-check/init` to get iProov token. cc @kober32